src/mbgl/renderer/tile_pyramid.hpp

```diff
@@ -42,7 +42,8 @@ class TilePyramid {
                 Range<uint8_t> zoomRange,
                 optional<LatLngBounds> bounds,
                 std::function<std::unique_ptr<Tile>(const OverscaledTileID&)> createTile,
-                optional<uint8_t> sourcePrefetchZoomDelta);
+                const optional<uint8_t>& sourcePrefetchZoomDelta,
+                const optional<uint8_t>& maxOverscaleFactor);
```

Comment:
same nit: `sourceMaxOverscaleFactor` ?

Reply:
tile pyramid is per source 1:1, I think source is superfluous here.
changed maxOverscaleFactor to maxParentTileOverscaleFactor